packages/react-dom/src/__tests__/ReactDOMOption-test.js

```diff
@@ -229,16 +247,27 @@ describe('ReactDOMOption', () => {
     expect(container.firstChild.getAttribute('value')).toBe(null);
     expect(container.firstChild.getAttribute('defaultValue')).toBe(null);
 
-    const option = container.firstChild.firstChild;
+    let option = container.firstChild.firstChild;
     expect(option.nodeName).toBe('OPTION');
 
     expect(option.textContent).toBe('BarFooBaz');
     expect(option.selected).toBe(true);
 
-    expect(() => ReactDOM.hydrate(children, container)).toErrorDev([
-      'Text content did not match. Server: "FooBaz" Client: "Foo"',
-      'validateDOMNesting(...): <div> cannot appear as a child of <option>.',
-    ]);
+    await expect(async () => {
+      await act(async () => {
+        ReactDOMClient.hydrateRoot(container, children);
+      });
+    }).toErrorDev(
+      [
+        'Warning: Text content did not match. Server: "FooBaz" Client: "Foo"',
+        'Warning: An error occurred during hydration. The server HTML was replaced with client content in <div>',
+        'Warning: validateDOMNesting(...): <div> cannot appear as a child of <option>',
+        'Error: Text content does not match server-rendered HTML.',
+        'Error: There was an error while hydrating. Because the error happened outside of a Suspense boundary, the entire root will switch to client rendering.',
+      ],
+      {withoutStack: 1},
+    );
+    option = container.firstChild.firstChild;
```

Comment:
The entire markup is now replaced with client rendering so we need to capture the newly created `option`

Reply:
Oh I guess I should actually read the comment huh